org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/linkedediting/LinkedEditingTest.java

```diff
@@ -141,13 +141,12 @@ protected boolean condition() {
 		}.waitForCondition(Display.getCurrent(), 3000);
 
 		IAnnotationModel model = sourceViewer.getAnnotationModel();
-		Annotation annotation = findAnnotation(sourceViewer, 14);
-		Assert.assertNotNull(annotation);
-//		System.out.println("testLinkedEditingExitPolicy: annotation found at offset 14: " + annotation.getType());
-		assertTrue(annotation.getType().startsWith("org.eclipse.ui.internal.workbench.texteditor.link"));
+		List<Annotation> annotations = findAnnotations(sourceViewer, 14);
+		Assert.assertFalse(annotations.isEmpty());
+		assertTrue(annotations.stream().anyMatch(a -> a.getType().startsWith("org.eclipse.ui.internal.workbench.texteditor.link")));
```

Comment:
better use `.filter(...).findAny()` here and `assertTrue(annotation.isPresent())`